gpytorch/lazy/lazy_tensor.py

```diff
@@ -898,6 +898,8 @@ def add_low_rank(
         # compute p = M B and take its SVD
         pvector = current_inv_root.matmul(low_rank_mat)
         # USV^T = p; when p is a vector this saves us the trouble of computing an orthonormal basis
+        if isinstance(pvector, LazyTensor):
+            pvector = pvector.evaluate()
```

Comment:
would suggest using `delazify` from `lazy.lazy_tensor` for this

Reply:
```suggestion
        pvector = delazify(pvector)
```